### ISSUE TITLE
How to change module to fix more image size.

### ISSUE THREAD

**Issue by maxh2010** [chenjun2hao/SRN.pytorch#21]
It seems to have resize the image to width to 100,.
if the image isn't resize to width to 100, the training process will report the RuntimeError.
The size pf tensor a (65) must match the size of tensor b (26) at non-singleton dimension 1

File SRN_modules.py line 68, in forward
    return x + self.pos_table[:, :x.size(1)].clone().detach()

How could i change to module to fit more kind of image size?

**Comment by maxh2010** [comment]
I can change position to 65, it will fix my issue
